src/SimpleCalculator.Web/wwwroot/js/site.js

```diff
@@ -1,4 +1,30 @@
-﻿// Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
-// for details on configuring this project to bundle and minify static web assets.
+﻿$(document).ready(function () {
+    _loadData(1);
+});
 
-// Write your JavaScript code.
+function _loadData(page) {
+    $.ajax({
+        url: 'https://localhost:7278/api/calculator?page=' + page,
```

Comment:
Please use a relative URL, that in case of localhost address is changed, your method still works.